cas-server-support-pac4j/pom.xml

```diff
@@ -70,7 +70,7 @@
 	</dependencies>
 
 	<properties>
-		<pac4j.version>1.6.0</pac4j.version>
+		<pac4j.version>1.7.0</pac4j.version>
```

Comment:
Could we externalize this into the parent pom? Trying to avoid phantom versions in specific poms.